azurerm/internal/services/cosmos/cosmosdb_gremlin_graph_resource.go

```diff
@@ -114,7 +114,7 @@ func resourceCosmosDbGremlinGraph() *schema.Resource {
 								string(documentdb.Consistent),
 								string(documentdb.Lazy),
 								string(documentdb.None),
-							}, false),
+							}, true),
```

Comment:
same here, why is this now case insensitive?

Reply:
same here. To solve the breaking change.

Reply:
same as other 2 - lets correc the case on read and add a comment to an open issue tracking this